stability-tests/reorg/reorg.go

```diff
@@ -19,6 +19,7 @@ import (
 func testReorg(cfg *configFlags) {
 	params := dagconfig.DevnetParams
 	params.SkipProofOfWork = true
+	params.DisableDifficultyAdjustment = true
```

Comment:
Why? If it's a bug please open an issue

Reply:
SkipProofOfWork without difficulty adjustment creates a ton of blocks with the same timestamp, which kills some assumption in the difficulty adjustment algorithm.